projects.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Why do you need this file, when index.html also contains the projects?

Reply:
oh i wanted the home page to list 4 of my newest projects and the project page to have a list of all my projects